lib/ansible_tower_client/job_template.rb

```diff
@@ -3,7 +3,7 @@ class JobTemplate < BaseModel
     def launch(options = {})
       launch_url = "#{url}launch/"
       options = options.dup
-      new_limit = options.delete('limit')
+      new_limit = options.delete('limit') { options.delete(:limit) }
```

Comment:
@syncrou For something this simple I think this reads better: 
`options.delete('limit') || options.delete(:limit)`  (If you are expecting a symbol to be passed from our code it would be good to swap the order as well.)

If you needed to use the block I like the idea of passing the key into and calling `to_sym`.
`options.delete('limit') { |key| options.delete(key.to_sym) }`